src/main/resources/data-repository-openapi.yaml

```diff
@@ -77,135 +73,214 @@ paths:
         500:
           description: service is broken
 
-  ## Access Interface
+  ## Repository Interface
 
-  '/api/access/v1/dataset':
-    get:
+  '/api/repository/v1/datasets':
+    get:  ## enumerate
       description: |
         Returns a list of all of the datasets the caller has access to
       operationId: enumerateDatasets
       tags:
-        - access
-      parameters:
-        - in: query
-          name: containing
-          type: string
-          description: return only datasets with name or description containing the string
+      - repository
       responses:
         200:
           description: List of datasets
           schema:
             type: array
             items:
-              $ref: '#/definitions/DatasetModel'
+              $ref: '#/definitions/DatasetSummaryModel'
 
-  ## Search Interface
-  '/api/search/v1/dataset':
-    get:
-      description: |
-        Given a search query in the body, returns a list of matching dataset ids
-      operationId: datasetSearch
+    post:
+      description: Create a new dataset
+      operationId: createDataset
+      tags:
+      - repository
+      parameters:
+      - in: body
+        name: dataset
+        description: Dataset to create
+        schema:
+          $ref: '#/definitions/DatasetRequestModel'
+      responses:
+        202: # kicks off a create dataset job
+          description: Dataset creation job started & url for polling is in the response header
+          headers:
+            location:
+              schema:
+                type: string
```

Comment:
you can't specify the schema for a non-custom header - remove schema and type  (i think 4 places in this file)